Tests/FluentAssertions.Specs/Execution/MessageFormatingSpecs.cs

```diff
@@ -99,52 +103,15 @@ public void When_an_assertion_fails_on_ContainSingle_succeeding_message_should_b
                 values.Should().ContainSingle();
             };
 
-            // Assert
-            act.Should().Throw<XunitException>()
-                .WithMessage("Expected*to contain a single item, but the collection is empty*" +
-                "Expected*to contain a single item, but the collection is empty*");
-        }
-
-        [Fact]
-        public void When_an_assertion_fails_on_BeOfType_succeeding_message_should_be_included()
```

Comment:
❓ Why were they removed?

Reply:
I couldn't think of a reason why this should be a test related to formatting. It is covered by type tests related to `BeOfType`.

Reply:
I can move them to the respective sections? I don't see why they were in the `Message Formatting` section of `AssertionScope`. But maybe it's the test name that is confusing me.

Reply:
They were probably put here because I did that for `ContainKey` and `ContainSingle` in b4d67280bd2f48fd9aa15125f1157e55f65d7955.

But you're right, they all four belong in their respective Specs files and not in formatting.

Reply:
I'll move the original tests around.